src/ceph-volume/ceph_volume/configuration.py

```diff
@@ -1,7 +1,10 @@
-try:
-    import configparser
```

Comment:
why isn't it is sufficient to swap the order of the imports? Why is it needed to check the version information here?

Reply:
Swapping would work too. But this way it is more clear (at least to me) why one expect an ImportError. It was hard to understand `import ConfigParser as configparser`.